<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Racing Car Game</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/src/icon.ico</normaloff>:/src/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>4800</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/map.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>384</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/game_name.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblCar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>237</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/car.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/src/start_btn.png</normaloff>:/src/start_btn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>250</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lblDistance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>300</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>行駛距離 : 0公尺</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdClock">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="block1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>290</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item1.png</pixmap>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>block5</zorder>
   <zorder>block4</zorder>
   <zorder>block3</zorder>
   <zorder>block2</zorder>
   <zorder>block1</zorder>
   <zorder>lblTitle</zorder>
   <zorder>lblCar</zorder>
   <zorder>buttonStart</zorder>
   <zorder>lblDistance</zorder>
   <zorder>lcdClock</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>選單</string>
    </property>
    <addaction name="action_Start_Game"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_Start_Game">
   <property name="text">
    <string>開始遊戲</string>
   </property>
   <property name="toolTip">
    <string>開始遊戲</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>關閉遊戲</string>
   </property>
   <property name="toolTip">
    <string>關閉遊戲</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
